ironfish-rust/src/keys/view_keys.rs

```diff
@@ -92,6 +92,58 @@ impl IncomingViewKey {
     }
 }
 
+#[derive(Clone)]
+pub struct ViewKey {
+    pub authorizing_key: jubjub::SubgroupPoint,
+    pub nullifier_deriving_key: jubjub::SubgroupPoint,
+}
+
+impl ViewKey {
+    /// Load a key from a string of hexadecimal digits
+    pub fn from_hex(value: &str) -> Result<Self, IronfishError> {
+        match hex_to_bytes(value) {
```

Comment:
I will put in subsequent PR to rename this error to `InvalidViewKey`, it is present in a few places and has incorrect tense.

Reply:
Nice, ty for suggestion